apps/sensenet/src/components/drawer/PermanentDrawer.tsx

```diff
@@ -1,43 +1,77 @@
-import IconButton from '@material-ui/core/IconButton'
 import List from '@material-ui/core/List'
 import ListItem from '@material-ui/core/ListItem'
 import ListItemIcon from '@material-ui/core/ListItemIcon'
-import ListItemSecondaryAction from '@material-ui/core/ListItemSecondaryAction'
 import ListItemText from '@material-ui/core/ListItemText'
 import Paper from '@material-ui/core/Paper'
 import Tooltip from '@material-ui/core/Tooltip'
-import KeyboardArrowLeft from '@material-ui/icons/KeyboardArrowLeft'
-import KeyboardArrowRight from '@material-ui/icons/KeyboardArrowRight'
-import Settings from '@material-ui/icons/Settings'
-import { PathHelper } from '@sensenet/client-utils'
+import { Close, Menu } from '@material-ui/icons'
 import React, { useContext, useEffect, useState } from 'react'
 import { withRouter } from 'react-router'
 import { matchPath, NavLink, RouteComponentProps } from 'react-router-dom'
-import { useRepository, useSession } from '@sensenet/hooks-react'
-import { ResponsiveContext, ResponsivePersonalSetttings } from '../../context'
-import { useDrawerItems, useLocalization, usePersonalSettings, useSelectionService, useTheme } from '../../hooks'
-import { LogoutButton } from '../LogoutButton'
-import { UserAvatar } from '../UserAvatar'
+import { useRepository } from '@sensenet/hooks-react'
+import clsx from 'clsx'
+import { createStyles, makeStyles, Theme } from '@material-ui/core'
+import { ResponsivePersonalSetttings } from '../../context'
+import { useDrawerItems, useLocalization, useSelectionService } from '../../hooks'
 import { AddButton } from '../AddButton'
 
+const useStyles = makeStyles((theme: Theme) => {
+  const primary = theme.palette.primary.main
```

Comment:
There are unused styles here. navLinkActiveStyle, listItemIconDark, listItemIconLight.
The primary variable is not needed. You can inline the value. Also because of the unused styles the implementation does not match the design.

Reply:
I see hover is now working for the collapse icon and for the add button but not for other elements. Please add that styling to those elements as well.